Wikibase/Tests/Commenting/DisallowedDocTags.php.expected

```diff
@@ -14,3 +14,4 @@
  45 | ERROR | [x] Found @Throw instead of @throws
  46 | ERROR | [x] Found @Type instead of @var
  47 | ERROR | [x] Found @Use instead of @uses
+ 68 | ERROR | [ ] PHPDoc tag @package is not allowed
```

Comment:
It would be nice if we could make this fixable as well… perhaps use `false` instead of `null` in `$disallowedTags` to mean “remove this tag without replacement”? But then we also need to determine that the whole doc comment is now empty, and remove it completely…

Reply:
I had the same idea, but realized removing such a tag is hard. What if the tag appears in the middle of a line of text? What do you remove then?

I will still try to make this happen, e.g. only remove a forbidden tag if the line starts with it.